<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPresentationBase</class>
 <widget class="QWidget" name="DlgPresentationBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>525</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Navigation</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="kcfg_SlidesAdvance">
          <property name="text">
           <string>Advance every:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPluralHandlingSpinBox" name="kcfg_SlidesAdvanceTime">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_SlidesLoop">
        <property name="text">
         <string>Loop after last page</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QLabel">
          <property name="text">
           <string>Touch navigation:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="kcfg_SlidesTapNavigation">
          <item>
           <property name="text">
            <string>Tap left/right side to go back/forward</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tap anywhere to go forward</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>Background color:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KColorButton" name="kcfg_SlidesBackgroundColor"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>Mouse cursor:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="kcfg_SlidesCursor">
          <item>
           <property name="text">
            <string>Hidden After Delay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always Visible</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always Hidden</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_SlidesShowProgress">
        <property name="text">
         <string>Show &amp;progress indicator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_SlidesShowSummary">
        <property name="text">
         <string>Show s&amp;ummary page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="kcfg_SlidesTransitionsEnabled">
     <property name="title">
      <string>Enable transitions</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Default transition:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_SlidesTransition">
        <item>
         <property name="text">
          <string>Blinds Vertical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blinds Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Box In</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Box Out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dissolve</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fade</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Glitter Down</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Glitter Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Glitter Right-Down</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random Transition</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Replace</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Split Horizontal In</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Split Horizontal Out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Split Vertical In</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Split Vertical Out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wipe Down</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wipe Right</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wipe Left</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wipe Up</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Placement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Screen:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="screenCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="annotationToolsGroupBox">
     <property name="title">
      <string>Drawing Tool Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>KPluralHandlingSpinBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_SlidesAdvance</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_SlidesAdvanceTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
